TACC/Core-CMS
# feat: tup-647 new getting_started template

## Overview

Add a v3 `getting-started.html` template.

## Status

> **Important**
> [Requested design review.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1699993116695629)

## Related

- [TUP-647](https://jira.tacc.utexas.edu/browse/TUP-647)

## Changes

- **fixed** an old guide template
- **added** a new guide template
- **added** settings for all guide templates

## Testing

0. Be logged in.
1. Open CMS.
2. Create/Edit a "Getting Started" page.
3. Edit template.
4. See 3 Getting Started template options:
    1. (v3)
    2. (TAM)
    3. (v2)
5. Choose (v3) template.
6. Review output, e.g.
    - images
    - layout
    - wording

## UI

| Template (Styles v3) | **New Template (Styles v2)** | [New Content Reference](https://cep.tacc.utexas.edu/guides/getting-started/) |
| - | - | - |
| ![guide - getting started](https://github.com/TACC/Core-CMS/assets/62723358/291c3920-0e35-45e4-863c-72d441840cf4) | ![guide - getting started v2](https://github.com/TACC/Core-CMS/assets/62723358/7f65a8f0-7dae-40de-95a1-89fd83ae7fc2) | ![v3 ref page](https://github.com/TACC/Core-CMS/assets/62723358/0b12b725-88ea-428b-a5b2-d8e32ba6a55d) |

<details><summary>Other Guides (Styles v2)</summary>

| Data Transfer | Globus Data | Portal Tech |
| - | - | - |
| ![guide - data transfer](https://github.com/TACC/Core-CMS/assets/62723358/dc87d314-9d94-4887-b85e-eab1679e58cd) | ![guide - globus data](https://github.com/TACC/Core-CMS/assets/62723358/590c87af-2805-45d7-be8e-bbe12d3450ed) | ![guide - portal tech](https://github.com/TACC/Core-CMS/assets/62723358/a0f691e5-73f8-4fc8-bf09-a0a5525e1c7e) |

</details>
